src/cryptonote_core/blockchain.cpp

```diff
@@ -2287,8 +2288,9 @@ bool Blockchain::check_tx_inputs(transaction& tx, uint64_t& max_used_block_heigh
     size_t ring_size = tx.vin[0].type() == typeid(txin_to_key) ? boost::get<txin_to_key>(tx.vin[0]).key_offsets.size() : 0;
     MINFO("HASH: " <<  get_transaction_hash(tx) << " I/M/O: " << tx.vin.size() << "/" << ring_size << "/" << tx.vout.size() << " H: " << max_used_block_height << " ms: " << a + m_fake_scan_time << " B: " << get_object_blobsize(tx));
 
-    if(ring_size > 16){
-      MERROR_VER("tx " << ring_size << " ring size greater than 16, rejecting tx");
+    //reject tx with mixin over 16 when blockheight is > 250000
+    if(ring_size > 16 && m_db->height() > 250000){
```

Comment:
Just a really minor comment: Would be good if you create a constant for these values in cryptonote_config file. I would suggest "MAX_RING_SIZE" and "MAX_BLOCKHEIGHT". 

Awesome work buddy!